cli/generate_operator_keys.go

```diff
@@ -15,6 +18,8 @@ var generateOperatorKeysCmd = &cobra.Command{
 	Use:   "generate-operator-keys",
 	Short: "generates ssv operator keys",
 	Run: func(cmd *cobra.Command, args []string) {
+		encryptionPassword, _ := cmd.Flags().GetString("password")
```

Comment:
password as parameter is unsafe because commands are remembered in terminal's history (in your ~/.bash_history or ~/.zsh_history files)

let's please accept `--password-file` parameter and read the password from it instead

another option is to request safe password input from user (in Go it's `term.ReadPassword`), but external scripts can't really use this